### ISSUE TITLE
Bug: Rename Files does not work

### ISSUE THREAD

**Issue by allkindsofralph** [evilhero/mylar#1751]
When I try to rename files, I only get an error that says "an error occurred"

**Comment by evilhero** [comment]
Unfortunately, I need more info than just 'an error occurred.'

What branch / commit of mylar are you running and are you up to date? 

What screen were you on when you did the rename action? What's your file format string set to? What's the series that you're trying to rename? 

Enable verbose logging and try to rename again and see if there's any additional information in the log.

**Comment by allkindsofralph** [comment]
I downloaded the master branch

version reads as: ca3c2cc28e2d07f05ff28d9d18bd5ee74fe239d5
I checked for update on link in bottom corner but nothing found

I am on the comic page for Invincible Iron Man trying to rename my files.

"An error occurred" appears in the bottom right corner

Format string: $Series $Issue ($Year)

Verbose Logging is on. I cleared the logs and tried to rename again but nothing recorded in the log.
